apps/challenges/models.py

```diff
@@ -504,6 +504,11 @@ class ChallengeConfiguration(TimeStampedModel):
         null=True,
         blank=True,
     )
+    code_upload_environment_log_file = models.FileField(
+        upload_to=RandomFileName("zip_configuration_files/challenge_zip"),
+        null=True,
+        blank=True,
+    )
```

Comment:
@matthew-so why do we need a log file in challenge model?
Can we remove this?

Reply:
It's a log file in the same way that stderr and stdout are. I named it this since it seemed cleaner than calling them code_upload_enverr. Is there a better name for it?